vercel.json

```diff
@@ -1,3 +1,9 @@
 {
-  "rewrites": [{ "source": "/(.*)", "destination": "/index.html" }]
-}
+  "buildCommand": "pnpm run build:generate-entry-points && vite build --mode staging && pnpm run build:inject-app-deeplinks",
```

Comment:
will adding this override the build command for our testnet environmet?

Reply:
By all means. The last commit was deliberetly added as a temporary measure, just in this PR, to see if the build would work if the command was adjusted. This was discussed with Jared out of band. Let me remove this change now.

Reply:
Commit removed.